qa-niaid.planx-pla.net/manifest.json

```diff
@@ -7,27 +7,27 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.04",
+    "arborist": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/arborist:2021.07",
     "audit-service": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/audit-service:1.0.0",
```

Comment:
why not audit-service too?

Reply:
Yes I wanted to ask couple of questions latest version of fence and make changes to both accordingly. Thank you for clarifying it on slack